Primo.js

```diff
@@ -363,37 +358,184 @@ function importPNX(text) {
+				"callNumber": "U21.2",
+				"language": "eng",
+				"libraryCatalog": "Primo",
+				"numPages": "xi+385",
+				"place": "Oxford ; New York",
+				"publisher": "Oxford University Press",
+				"attachments": [],
 				"tags": [
-					"Thelen, Albert Vigoleis, 1903-1989 Y"
+					"War."
 				],
-				"seeAlso": [],
+				"notes": [],
+				"seeAlso": []
+			}
+		]
+	},
+	{
+		"type": "web",
+		"url": "http://limo.libis.be/primo_library/libweb/action/dlDisplay.do?vid=LIBISnet&docId=32LIBIS_ALMA_DS71166851730001471&fn=permalink",
+		"items": [
+			{
+				"itemType": "book",
+				"title": "War",
+				"creators": [
+					{
+						"firstName": "Albert R.",
+						"lastName": "Leventhal",
+						"creatorType": "author"
+					},
+					{
+						"firstName": "Del",
+						"lastName": "Byrne",
+						"creatorType": "contributor"
+					}
+				],
+				"date": "1973",
+				"ISBN": "0600393046",
+				"language": "eng",
+				"libraryCatalog": "Primo",
+				"numPages": "252",
+				"publisher": "Hamlyn",
 				"attachments": [],
-				"title": "In Zweifelsfällen entscheidet die Wahrheit : Beiträge zu Albert Vigoleis Thelen",
-				"place": "Viersen",
-				"publisher": "JUNI",
-				"date": "1988",
-				"language": "ger",
-				"numPages": "149",
-				"ISBN": "3926738014",
+				"tags": [],
+				"notes": [],
+				"seeAlso": []
+			}
+		]
+	},
+	{
+		"type": "web",
+		"url": "http://virtuose.uqam.ca/primo_library/libweb/action/dlDisplay.do?vid=UQAM&docId=UQAM_BIB000969205&fn=permalink",
+		"items": [
+			{
+				"itemType": "book",
+				"title": "War",
+				"creators": [
+					{
+						"firstName": "Ken",
+						"lastName": "Baynes",
+						"creatorType": "author"
+					},
+					{
+						"lastName": "Welsh Arts Council",
+						"creatorType": "contributor",
+						"fieldMode": 1
+					},
+					{
+						"lastName": "Glynn Vivian Art Gallery",
+						"creatorType": "contributor",
+						"fieldMode": 1
+					}
+				],
+				"date": "1970",
+				"language": "eng",
 				"libraryCatalog": "Primo",
-				"shortTitle": "In Zweifelsfällen entscheidet die Wahrheit"
+				"publisher": "Boston Boston Book and Art Chop",
+				"attachments": [],
+				"tags": [
+					"ART",
+					"GUERRE",
+					"WAR"
+				],
+				"notes": [],
+				"seeAlso": []
```

Comment:
This book is part of a book series. Maybe we could add that information as well from:
```
<prim:seriestitle>Art and society 1</prim:seriestitle> 
```